File: SpringBootBackend/src/main/java/com/th/dto/VendorDTO.java
package com.th.dto;

import java.math.BigDecimal;

import com.th.entity.Vendor.Status;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class VendorDTO extends BaseDTO {
    private String companyName;
    private String email;
    //private String password;
    private String phone;
    private String gstin;
    private String companyOwnerName;
    private String ownerAadharNumber;
    private String panNumber;
    private String city;
    private BigDecimal amount;
    private Status status;
    
   
}

File: DOTNET BACKEND/TransitHive/Dtos/VendorDto.cs
using TransitHive.Models;

namespace TransitHive.Dtos
{
    public class VendorDto
    {
        public int Id { get; set; }
        public string CompanyName { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Phone { get; set; }
        public string Gstin { get; set; }
        public string CompanyOwnerName { get; set; }
        public string OwnerAadharNumber { get; set; }
        public string PanNumber { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public string Amount { get; set; }
        public Status Status { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
    }

  

}
